prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -52,7 +52,10 @@ presubmits:
       containers:
       - image: alpine:3.15.0
```

Comment:
That is a good point, after trying gcloud, I found that it doesn't have `apt` which is required by presubmit.sh. So I will just use alpine because it will install bash if not exist.

Reply:
I'm not sure if alpine comes with `apt` by default. Looks like you need a not-so-minimum image, I'm guessing you might need to try debian slim or something similar.

Reply:
Sorry! I have updated now.

Reply:
I am able to use `apt` for alpine, or should I try debian slim? Will that be too big for presubmit run?

Reply:
No, if it works, then never mind. :) Let's merge and try it out.